web/src/modules/layout/components/HeaderActionItemDropDown.js

```diff
@@ -42,16 +42,14 @@ export const DropDownContainer = styled.div`
   right: 0;
   width: 100%;
   box-sizing: border-box;
-  opacity: ${props => props.active ? '1' : '0'};
   z-index: 1; /* this is only necessary for IE11 to have the DropDown above NavigationItems */
 
-  transform: scale(${props => props.active ? '1' : '0.9'});
+  transform: scale(1);
```

Comment:
I think `scale(1)` has no effect, meaning it can be removed (https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/scale`.
~~Why is this (and opacity and visibility) not required anymore?~~